guides/support_playbook.md

```diff
@@ -114,10 +114,10 @@ The example below copies scheduled courses during rollover from one provider to
 
 ```ruby
 # Find provider to copy courses from
-provider = RecruitmentCycle.current.next.providers.find_by(provider_code: "1TZ")
```

Comment:
This was probably written when the two cycles are open (during rollover) when there would be a next recruitment cycle.